### PR TITLE
Add withdrawn projects to SPP interconnection data

### PR DESCRIPTION
**Pull request by bendnorman**
This PR:

- Updates the SPP interconnection queue to include withdrawn projects
- Fixes a bug in the NYISO interconnection queue cleaning that resulted in missing developer information.

### CODE REVIEW

File: gridstatus/nyiso.py [hunk 1]
```diff
@@ -541,7 +543,7 @@ def get_interconnection_queue(self, verbose=False):
             "Project Name": "Project Name",
             "County": "County",
             "State": "State",
-            "Owner/Developer": "Interconnecting Entity",
+            "Developer Name": "Interconnecting Entity",
```
Comment (bendnorman) [inline gridstatus/nyiso.py:546]: In the raw data, only the "Interconnection Queue" tab has a "Developer Name" column. The "Owner/Developer" column was renamed in the `completed` dataframe but not the `withdrawn` dataframe.

File: gridstatus/spp.py [hunk 2]
```diff
@@ -694,7 +694,7 @@ def _get_load_and_forecast(self, verbose=False):
     # 15mb file with five minute resolution
 
     def get_raw_interconnection_queue(self, verbose=False) -> BinaryIO:
-        url = "https://opsportal.spp.org/Studies/GenerateActiveCSV"
+        url = "https://opsportal.spp.org/Studies/GenerateSummaryCSV"
```
Comment (bendnorman) [inline gridstatus/spp.py:697]: I found another [url](https://opsportal.spp.org/Studies/GIRequests) on the SPP website that includes all withdrawn projects:

image

Comment (WillKoehrsen) [reply gridstatus/spp.py:697]: Good find! Can we also update `get_interconnection_queue` so the "WITHDRAWN" status maps to `InterconnectionQueueStatus.WITHDRAWN.value`?

Comment (bendnorman) [reply gridstatus/spp.py:697]: Ah yes thank you! I just mapped "WITHDRAWN" to the `InterconnectionQueueStatus.WITHDRAWN.value` and added the withdrawn date. About a third of withdrawn projects do not have a withdrawn date. The vast majority of withdrawn projects missing a withdrawn date are from the early 2000s.

**Comment by bendnorman** [comment]
Is the CI failing because this external PR does not have access to repo secrets?

**Comment by WillKoehrsen** [comment]
> Is the CI failing because this external PR does not have access to repo secrets?

Yeah. I pulled your code and ran `test_nyiso.py` and `test_spp.py` (the tests potentially affected by this PR) and they passed.

**Comment by bendnorman** [comment]
Great thank you!

**Pull request merged**